config/node1.toml

```diff
@@ -25,7 +25,7 @@ disable = true
 enable = false
 
 [account]
-unlock = ["0xbbcaa8d48289bb1ffcf9808d9aa4b1d215054c78", "0x32e4e4c7c5d1cea5db5f9202a9e4d99e56c91a24"]
+unlock = ["0xbbcaa8d48289bb1ffcf9808d9aa4b1d215054c78", "0x32e4e4c7c5d1cea5db5f9202a9e4d99e56c91a24", "0xf67cc5231c5858ad6cc87b105217426e17b824bb", "0xbe69eb0968226a1808975e1a1f2127667f2bffb3", "0x720e118ab1006cc97ed2ef6b4b49ac04bb3aa6d9"]
```

Comment:
Here are too much unlocked accounts. Let's be close to production mode and unlock in `toml` only those accounts which need to be unlocked for properly work of a node.

It's better to unlock other accounts by `web3` functions. Or use `keythereum` npm package to work with private key imported from json keystore file and do transactions manually with `ethereumjs-tx` npm package like in https://github.com/poanetwork/poa-network-consensus-contracts/blob/862c6e7b48f7a803e0282f05e90287b70b32af2a/scripts/migrate/utils/utils.js#L47-L74

We'll need soon to unlock other accounts due to https://github.com/poanetwork/parity-ethereum/issues/97

So it will be a tangle of unlocked accounts and will confuse in the end.